### PR TITLE
[PiranhaJS] Updated links in package.json

### PR DESCRIPTION
**Pull request by mkr-plse**
In the `package.json` file in the `javascript` directory, the links were pointing to the forked location. Updated the links.

### CODE REVIEW

File: javascript/package.json [hunk 1]
```diff
@@ -13,15 +13,15 @@
...
     },
     "keywords": [],
     "author": "",
     "license": "ISC",
     "bugs": {
-        "url": "https://github.com/anag004/piranhaJS/issues"
+        "url": "https://github.com/uber/piranha/issues"
     },
-    "homepage": "https://github.com/anag004/piranhaJS#readme",
+    "homepage": "https://github.com/uber/piranha#readme",
```
Comment (lazaroclapp) [inline javascript/package.json:24]: Should this one be the global Piranha README or https://github.com/uber/piranha/blob/master/javascript/README.md ?

One can argue either way, and I am fine with going to top-level, specially for a link called `homepage`, but I thought I'd bring up the choice :)

**Comment by lazaroclapp** [comment]
Oh, and please remember to land as `"[PiranhaJS] Updated links in package.json"`

File: javascript/package.json (hunk 1 above)
Comment (mkr-plse) [reply javascript/package.json]: Since JS README was linked from homepage, I used that. Changed it now.

**Comment by mkr-plse** [comment]
> Oh, and please remember to land as `"[PiranhaJS] Updated links in package.json"`

Done :)

**Pull request merged**